src/ucp/core/ucp_context.c

```diff
@@ -1159,16 +1164,19 @@ ucp_add_tl_resources(ucp_context_h context, ucp_md_index_t md_index,
                      uint64_t *tl_cfg_mask)
 {
     ucp_tl_md_t *md = &context->tl_mds[md_index];
-    uct_tl_resource_desc_t *tl_resources;
+    uct_tl_resource_desc_v2_t *tl_resources;
```

Comment:
seems we can use old API here

Reply:
we need v2 because we call `ucp_add_tl_resource_if_enabled()` on the end of the funcction.